guides/common/modules/proc_renewing-a-custom-ssl-certificate.adoc

```diff
@@ -0,0 +1,57 @@
+[id="Renewing_a_Custom_SSL_Certificate_{context}"]
+= Renewing a {customssltitle} certificate on {ProjectServer}
+
+Use this procedure to update your renewed {customssl} certificate signed by a Certificate Authority (CA) for {ProjectServer}.
+The `katello-certs-check` command validates the input certificate files and returns the commands necessary to renew a {customssl} certificate for {ProjectServer}.
+
+.Prerequisites
+Refer to the {Project} installation documentation before creating a new CSR because the Server certificate should have X.509 v3 `Key Usage` and `Extended Key Usage` extensions with required values.
+
+.Procedure
+You must create a new Certificate Signing Request (CSR) and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {ProjectServer} certificate and CA bundle.
+
+==  Creating a {customssltitle} for {ProjectServer}
+
+Before deploying a renewed custom certificate on your {ProjectServer}, complete the following steps:
+
+* Validate the {customssl} input files.
+Note that for the `katello-certs-check` command to work correctly, Common Name (CN) in the certificate must match the FQDN of {ProjectServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check \
+-c /root/satellite_cert/New_satellite_cert.pem \
```

Comment:
```suggestion
-c /root/{smart-proxy-context}_cert/new_{smart-proxy-context}_cert.pem \
```

same below

Reply:
Yes, my bad :facepalm:

Reply:
Actually, I should cover this file in the last commit itself. 🙈